lib/src/block_parser.dart

```diff
@@ -1227,3 +1232,31 @@ class ParagraphSyntax extends BlockSyntax {
     return true;
   }
 }
+
+/// Walks the parser forward through the lines do not match any [BlockSyntax].
```

Comment:
grammar nit, something like "... through the lines; does not ..."

Reply:
Fixed